tests/test.c

```diff
@@ -603,11 +603,25 @@ void same_param_value(char* dev_name, uint64_t uid, char* param_name, param_type
 // Returns if value is between the expected high and expected low
 void check_param_range(char* dev_name, uint64_t uid, char* param_name, param_type_t param_type, param_val_t expected_low, param_val_t expected_high) {
     uint8_t dev_type = device_name_to_type(dev_name);
+    if(dev_type == -1){
+        print_fail();
+        fprintf(stderr, "%s type not found\n", dev_name);
+        end_test();
+        exit(1);
+    }
     device_t* dev = get_device(dev_type);
-
+    if(dev == NULL){
```

Comment:
this should never fail since the previous `dev_type` isn't -1

Reply:
You removed it in `check_latency`, but it also can be removed here in `check_param_range`.